usmqe_tests/tendrlapi/test_volume.py

```diff
@@ -3,37 +3,35 @@
 """
 import pytest
 
-import json
-
-import re
-
 from usmqe.api.tendrlapi import tendrlapi
 from usmqe.gluster import gluster
 from usmqe.api.etcdapi import etcdapi
+import usmqe.inventory as inventory
 
 
 @pytest.fixture
 def cluster_id():
     api = tendrlapi.ApiGluster()
-    response = api.call(pattern="GetClusterList", method="GET")
-
-    expected_response = 200
-    pytest.check(response.status_code == expected_response)
-
-    return response.json()[0]["cluster_id"]
+    return api.get_cluster_list()[0]["cluster_id"]
 
 
 @pytest.fixture
-def volume_id():
-    test_gluster = gluster.GlusterCommon()
-    xml = test_gluster.run_on_node(command="volume info")
-    return xml.findtext("./volInfo/volumes/volume/id")
+def volume_id(cluster_id):
+    api = tendrlapi.ApiGluster()
+    volumes = api.get_volume_list(cluster_id)
+    volume_id = False
+    for item in volumes:
+        if item["name"] == "Vol_test":
```

Comment:
Is this testcase bound with other testcase where **"Vol_test"** is created? I think **"Vol_test"** should be in configuration.